### PR TITLE
feat(issue-views): Update banner and tooltip copy, make banner dismissable, feedback -> read docs

### PR DESCRIPTION
**Pull request by MichaelSun48**
Makes a couple changes to the add view page for EA readiness: 

1. Updates the banner copy to call out: views are user scoped, views are reorderable, and that custom searches are persisted temporarily (only appears if you actually have any custom searches)
2. Adds a dismiss button to the banner 
3. Switches the "Give Feedback" button for a "Read Docs" button 
4. Adds a "Learn More" link to docs within the saved searches tooltip 

**IMPORTANT**: Issue views docs have not been merged in yet, so the docs links are **broken**. I will not merge this until the docs links resolve. [Docs PR](https://github.com/getsentry/sentry-docs/pull/11390) if you're curious about it.

### CODE REVIEW

File: static/app/views/issueList/addViewPage.tsx [hunk 1]
```diff
@@ -92,6 +93,67 @@ function AddViewPage({savedSearches}: {savedSearches: SavedSearch[]}) {
...
+
+  return !isPromptDismissed ? (
+    <Banner>
+      <BannerStar1 src={bannerStar} />
+      <BannerStar2 src={bannerStar} />
+      <BannerStar3 src={bannerStar} />
+      <Title>
+        {t('Welcome to the new Issue Views experience (Early Adopter only)')}
+        <DismissButton
+          analyticsEventKey="issue_views.add_view.banner_dismissed"
```
Comment (roggenkemper) [inline static/app/views/issueList/addViewPage.tsx:112]: if the button is the only place you are using this analytics event key, you don't need to define it in `issueAnalyticsEvents`

File: static/app/views/issueList/addViewPage.tsx [hunk 2]
```diff
@@ -92,6 +93,67 @@ function AddViewPage({savedSearches}: {savedSearches: SavedSearch[]}) {
   );
 }
 
+function AddViewBanner({hasSavedSearches}: {hasSavedSearches: boolean}) {
+  const organization = useOrganization();
+
+  const {isPromptDismissed, dismissPrompt} = usePrompt({
+    feature: 'issue_views_add_view_banner',
```
Comment (malwilley) [inline static/app/views/issueList/addViewPage.tsx:100]: Have you added this to the backend? There should be a list of constants that the backend will validate from

File: static/app/views/issueList/addViewPage.tsx [hunk 3]
```diff
@@ -92,6 +93,67 @@ function AddViewPage({savedSearches}: {savedSearches: SavedSearch[]}) {
...
+            <b>Views are for your eyes only.</b> No need to worry about messing up other
+            team members' views
+          </li>
+          <li>
+            <b>Drag your views to reorder.</b> The leftmost view is your “default”
+            experience
+          </li>
+          {hasSavedSearches && (
+            <li>
+              <b>Custom searches will be deprecated in the future. </b> You can save them
```
Comment (malwilley) [inline static/app/views/issueList/addViewPage.tsx:139]: We refer to them as "custom" searches here, but "saved" in the header below

File: static/app/views/issueList/addViewPage.tsx (hunk 2 above)
Comment (MichaelSun48) [reply static/app/views/issueList/addViewPage.tsx:100]: Ah I did not, making a PR for this now

**Review by malwilley — APPROVED** [review]

**Pull request merged**